.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
       "request": "launch",
       "name": "Mocha All within 'test' folder",
       "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-      "args": ["--timeout", "999999", "--colors", "${workspaceFolder}/test/**/*"],
+      "args": ["--timeout", "999999", "--require", "${workspaceFolder}/test/mochaFixtures.js", "--colors", "${workspaceFolder}/test/**/*"],
```

Comment:
is this to link the new mochaFixtures functions with the test files when we run it through the VSCode debugger?

Reply:
Yes, that's right.